### PR TITLE
Adds the cfp blog post

### PR DESCRIPTION
**Pull request by harisibrahimkv**

### CODE REVIEW

**Comment by vnbang2003** [comment]
@realslimshanky I don't think you are part of volunteering Team for this year so please stop reviewing it.

**Comment by vnbang2003** [comment]
@realslimshanky We welcome everyone.
Can you help us understand if you were involved in discussion to approve the PR ?
Did you check with existing volunteer team on requirement atleast ?

**Comment by vnbang2003** [comment]
@harisibrahimkv  Can you check if we can add more details like what US PyCon CFP blog was
https://pycon.blogspot.in/2017/09/pycon-2018-call-for-proposals-is-open.html

File: content/blog/cfp-2018.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Announcing the Call For Proposals
+#################################
+
+:Date: 2018-01-12 18:05
+:Category: 2018
+
+No matter who you are, whatever you do, please - consider submitting a proposal!
```
Comment (ofpiyush) [inline content/blog/cfp-2018.rst]: the `-` can be dropped.

File: content/blog/cfp-2018.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+`Head over to this years junction and submit one right away.
```
Comment (ofpiyush) [inline content/blog/cfp-2018.rst]: `s/years/year's`

**Comment by ananyo2012** [comment]
#130 made for announcing CFP. Closing this one.

**Pull request closed without merging**